app/lib/features/onboarding/pages/start_page.dart

```diff
@@ -299,15 +298,17 @@ class _StartPageState extends State<StartPage> {
                             onBoardingPages[index].image,
                           ),
                         ),
+                        const SizedBox(
```

Comment:
I have no idea if that fixed it. Can you provide a screenshot?

Reply:
I included the screenshots in the respective issues but I I will included them here too .